frontend/src/components/Gym/ConfirmationModal.tsx

```diff
@@ -0,0 +1,124 @@
+import React from 'react'
+import Button from '../Button'
+import styled from 'styled-components'
+
+import 'antd/dist/antd.css'
+
+const OverlayContainer = styled.div<{ overlayBackground: string; isModalOpen: boolean }>`
```

Comment:
consider updating the name to something more intuitive, like `overlayBackgroundColor` instead?